app_check/integration_test/functions/firebase.json

```diff
@@ -0,0 +1,4 @@
+{
+    "functions": {
+    }
```

Comment:
What is this file? Can it be deleted?

Reply:
I assumed it wasn't needed, but when I deleted it, the deploy failed, complaining about this file missing.  From the functions samples, it has logic like https://github.com/firebase/functions-samples/blob/main/2nd-gen/callable-functions/firebase.json, but we didn't need that additional logic for this simple function, hence why it is sorta empty.

Reply:
Alright, keep it.